### ISSUE TITLE
Sails Lift with replica set (Primary down)

### ISSUE THREAD

**Issue by kdoopadahalli81** [balderdashy/sails-mongo#213]
Hi Team,

I'm new to sails, was working on mongodb replica set. This is a feature question rather than issue.

I was able to hit secondary when primary is down and get the data. My question is will we be able to lift sails when primary is down. Is there an option to do that.

**Comment by andypham** [comment]
@kdoopadahalli81 Set the below setting
replSet: {                                                                                                                                                                                                                                       
      connectWithNoPrimary: true                                                                                                                                                                                                                                          
    }

**Comment by sailsbot** [comment]
Thanks for posting, @kdoopadahalli81.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
